internal/tests/cli/boundary/_target_host_sources.bash

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+load _authorized_actions
+
+function set_target_host_sources() {
+    for i in "${@:2}"; do
+        hostSource+="-host-source $i "
+    done
+
+    boundary targets set-host-sources -id $1 $hostSource
+}
+
+function add_target_host_sources() {
+    for i in "${@:2}"; do
+        hostSource+="-host-source $i "
+    done
+
+    boundary targets add-host-sources -id $1 $hostSource
+}
+
+function remove_target_host_sources() {
+    for i in "${@:2}"; do
+        hostSource+="-host-source $i "
+    done
+
+    boundary targets remove-host-sources -id $1 $hostSource
+}
+
+function target_has_host_source_id() {
+    local tid=$1
```

Comment:
For clarity, I might suggest to also add variables for the input arguments here
```
local tid=$1
local hostSources...  # I'm not sure how to handle the array
local format=$3  # I think I preferred format being the last option
```

Reply:
It's trickier to have `hostSources` as the 2nd argument because it supports multiple host-sources. I've seen a pattern in other test suites to have the array at the end.
I can set `format` as a variable at the top